handbook/ops/bizops/customer_data_policy.md

```diff
@@ -1,15 +1,49 @@
-| Slack                                                                | Customer support channels are maintained indefinitely; all other messages are deleted after 3 months                                                                                                                      | Customer support channels are kept for context and to better diagnose future support issues                                                                                                            |
-| Google Cloud Platform data pipeline \(buckets, Dataflow, Pub/Sub\) | Pub/Sub and Dataflow don't hold data \(only manage the data pipelines\)\. GCP buckets follow the same rules as the data they backup \(i\.e\. Sourcegraph\.com data backed up in buckets is kept for a year\) |                                                                                                                                                                                                |
-| Communication tools \(Gmail, Jira, Productboard\)                  | Retain all inbound customer communication                                                                                                                                                                    |        For context with customer conversations and to inform future support issues and product development decisions                                                                                                                                                                                        |
-| [View list of sales tools](../../sales/onboarding.md#getting-started)                                                        | Retain all outbound customer communication                                                                                                                                                             |  For context with prospect conversations and do not contact lists
+
+PII and user characteristics
+| Source/platform                                   | Sourcegraph Cloud  | On-prem/self-hosted | How long is this data kept for?                                                                      | Who can access it? |
+|---------------------------------------------------|--------------------|---------------------|------------------------------------------------------------------------------------------------------|--------------------|
+| HubSpot and Salesforce (CRM/marketing automation) | :heavy-check-mark: | :heavy-check-mark:  | Retain all customer information                                                                      |                    |
```

Comment:
I know no changes were made to this table, but would it be useful to distinguish between Cloud and on-prem for this? 

For example, on HubSpot, we do collect information for _all_ authenticated users for Cloud. However, for on-prem, we don't actually collect any user information unless they complete an NPS survey, fill out a feedback form, etc.

Also it's a bit confusing that the section is PII and user characteristics, but we say we retain all *customer* information. For on-prem, customer and user are two different entities. While we collect customer info (company name, # users, etc.) we almost never collect identifying info for users (as outlined in the last paragraph).

Reply:
good call. how about this? https://github.com/sourcegraph/about/pull/3697/commits/05a7979260554c9344fa04f5beae79429093847d